### DISCUSSION TITLE
i forgot my user password

### DISCUSSION THREAD

**Discussion by minekraftgamer** [supabase/supabase#29726] [Questions]
i forgot my user password but i have my org password can i use that to add tables to the database using supabase-py

**Comment by GaryAustin1** [comment]
Can you be more specific on what password you lost and what org password is (not the value)?

**Reply by GaryAustin1** [reply]
I'm going to guess as you are still not clear below.
You forgot your database password?  If so you can reset that under the database settings.
![image](https://github.com/user-attachments/assets/e146b6c4-f2fe-4c49-abee-2b3012b93bfe)

If you mean you don't have the dashboard password... Then with Github login you would not have a password and if using email you should be able to reset it.

**Comment by minekraftgamer** [comment]
the account user password and the org password is the project password i guess
